src/app/Dashboard/DashboardCardActionMenu.tsx

```diff
@@ -64,9 +67,9 @@ export const DashboardCardActionMenu: React.FunctionComponent<DashboardCardActio
         onSelect={onSelect}
         dropdownItems={[
           <DropdownItem key="Remove" onClick={props.onRemove}>
-            Remove
+            {t('REMOVE', { ns: 'common' })}
           </DropdownItem>,
-        ]}
+        ].filter((item) => item !== null)}
```

Comment:
I think React will ignore `null` for us already? Or is it for any other purposes?

https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

Reply:
This was added before because I had a conditional in there that may have added a null component array item. That can't happen anymore so I'll remove it, thanks.